charts/validator/templates/deployment.yaml

```diff
@@ -33,12 +33,28 @@ spec:
           hostNetwork: true
           {{- end }}
           image: "{{ .Values.ghost.image.repository }}:{{ .Values.ghost.image.tag | default .Chart.AppVersion }}"
+          {{- if .Values.ghost.commandOverride }}
+          command:
+          {{- range $key, $val := .Values.ghost.commandOverride }}
+            - {{ $val }}
+          {{- end }}
+          {{- end }}
           args:
+          {{- if .Values.ghost.argsOverride }}
+          {{- range $key, $val := .Values.ghost.argsOverride }}
+            - {{ $val }}
+          {{- end }}
+          {{- else }}
             - "run"
             - "-v"
             - "{{ .Values.ghost.logLevel | default "warning" }}"
             - "--log.format"
             - "{{ .Values.ghost.logFormat | default "text" }}"
```

Comment:
This overrides image defaults.
I'd prefer:
```suggestion
```

Reply:
this should be in `env:` 
```
            - name: CFG_LOG_LEVEL
              value: {{ .Values.ghost.logLevel | default "debug" | quote }}
            - name: CFG_LOG_FORMAT
              value: {{ .Values.ghost.logFormat | default "text" | quote }}
```

Reply:
added this config as env vars instead of args